src/Sulu/Bundle/ContentBundle/Resources/public/js/main.js

```diff
@@ -37,6 +39,8 @@ define([
 
             'use strict';
 
+            SeoTab.initialize(app);
```

Comment:
`app.use` can only be called before start is done. this is the only way to not include the code in this extension file.